databox-test

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+RED='\033[0;31m'
+GREEN='\033[0;32m' 
+NC='\033[0m'
```

Comment:
definitely factor these out into some utility library to be `source`d I think

Reply:
done